pkg/config/run.go

```diff
@@ -25,20 +25,6 @@ type Run struct {
 
 	ShowStats bool `mapstructure:"show-stats"`
 
-	// --- Flags only section.
-
-	IsVerbose bool `mapstructure:"verbose"` // Flag only
-
-	PrintVersion bool // Flag only. (used by the root command)
-
-	CPUProfilePath string // Flag only.
-	MemProfilePath string // Flag only.
-	TracePath      string // Flag only.
```

Comment:
Didn't see any comment on this but I see you ripped out all the profiling parts. Just because it wasn't used and mostly aims for development of `golangci-lint`?

Reply:
It didn't remove the profiling, I just moved the profiling to the only command that needs profiling: the `run` command.

Reply:
Am I blind? I can't see the flags `cpu-profile-path` and `mem-profile-path` added back in any of your additions?

Reply:
`setupRunPersistentFlags` in the `run` command

Reply:
Omg so silly, missed it was collapsed and tried to sesrch for the term in the browser. 🤦🏽‍♀️ Sorry about that!